# Array of real musical tones
# The description of a tone could be find MusicalTones.msg
MusicalTones[] score
